pkg/store/bucket.go

```diff
@@ -169,6 +169,8 @@ type BucketStore struct {
 
 	// Verbose enabled additional logging.
 	debugLogging bool
+	// Number of goroutines to use when syncing blocks from object storage
```

Comment:
missing trailing dot in comment ^^ (sorry for being strict!)
```suggestion
	// Number of goroutines to use when syncing blocks from object storage.
```